src/Engine/ProtoCore/Parser/Parser.cs

```diff
@@ -1187,7 +1191,7 @@ private bool IsNotAttributeFunction()
 		node = null; 
 		Associative_LogicalExpression(out node);
 		while (la.kind == 53) {
-			Associative_TernaryOp(ref node);
+			AssociativeTernaryOp(ref node);
```

Comment:
You may revert the change in associative and imperative atg and regenerate Parser.cs